src/Sentry.Unity.Editor/ConfigurationWindow/SentryEditorWindowInstrumentation.cs

```diff
@@ -4,7 +4,7 @@
 using UnityEditor;
 using UnityEngine;
 
-namespace Sentry.Unity.Editor
+namespace Sentry.Unity.Editor.ConfigurationWindow
```

Comment:
line 170 of Directoryu.Build.Targets needs to be changed for:
```xml
    <Exec Command="&quot;$(UnityExec)&quot; -quit -batchmode -nographics -logFile - -projectPath $(UnitySampleProjectPath) -executeMethod Sentry.Unity.Editor.ConfigurationWindow.SentryEditorWindowInstrumentation.ConfigureOptions -sentryOptions.Dsn $(TestDsn)" IgnoreStandardErrorWarningFormat="true">
        <Output TaskParameter="ExitCode" PropertyName="UnityConfigureSentryOptionsExitCode"/>
    </Exec>
```

Reply:
thanks for pointing that out!